product_docs/docs/transporter/ba_preview/getting_started/installing/linux_x86_64/index.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: "Installing EDB Transporter on Linux x86 (amd64)"
```

Comment:
@Xander-run let's change the name of the titles through all operating systems to: 
```suggestion
title: "Installing EDB DMS Reader on Linux x86 (amd64)"
```